MUXControls.sln

```diff
@@ -1082,6 +1082,7 @@ Global
 		{FBC396F5-26DD-4CA3-981E-C7BC9FEA4546}.Debug|ARM.Build.0 = Debug|ARM
 		{FBC396F5-26DD-4CA3-981E-C7BC9FEA4546}.Debug|ARM.Deploy.0 = Debug|ARM
 		{FBC396F5-26DD-4CA3-981E-C7BC9FEA4546}.Debug|arm64.ActiveCfg = Debug|arm64
+		{FBC396F5-26DD-4CA3-981E-C7BC9FEA4546}.Debug|arm64.Build.0 = Debug|arm64
```

Comment:
What about arm64 release?

Reply:
What about the Debug_Test configuration?

Reply:
i reverted this change because of a build break. I'll add for debug_test when i redo it.

---
In reply to: [255692646](https://github.com/Microsoft/microsoft-ui-xaml/pull/288#discussion_r255692646) [](ancestors = 255692646)